src/main/java/stsjorbsmod/relics/BottledBurdenRelic.java

```diff
@@ -0,0 +1,153 @@
+
+    public BottledBurdenRelic(){super(ID, CULL_CARD_COLOR, RelicTier.COMMON, LandingSound.CLINK);}
+
+    @Override
+    public Predicate<AbstractCard> isOnCard() {
+        return inBottledBurden::get;
+    }
+
+    @Override
+    public Integer onSave() {
+        if (card != null) {
+            return AbstractDungeon.player.masterDeck.group.indexOf(card);
+        } else {
+            return -1;
+        }
+    }
+
+    @Override
+    public void onLoad(Integer cardIndex) {
+        if (cardIndex == null) {
+            return;
+        }
+        if (cardIndex >= 0 && cardIndex < AbstractDungeon.player.masterDeck.group.size()) {
+            card = AbstractDungeon.player.masterDeck.group.get(cardIndex);
+            if (card != null) {
+                inBottledBurden.set(card, true);
+                AbstractCard cardInDeck = AbstractDungeon.player.masterDeck.getSpecificCard(this.card);
+                EntombedField.entombed.set(cardInDeck, true);
+                setDescriptionAfterLoading();
+            }
+        }
+    }
+
+    @Override
+    public void onEquip() {
+        // follow same behavior as base game bottle relics.
+        CardGroup nonExertCards = CardUtils.getCardsForBottling(AbstractDungeon.player.masterDeck.getPurgeableCards());
+        nonExertCards.group.removeIf(c -> SelfExertField.selfExert.get(c));
+        if (nonExertCards.size() > 0) {
+            this.cardSelected = false;
+            if (AbstractDungeon.isScreenUp) {
+                AbstractDungeon.dynamicBanner.hide();
+                AbstractDungeon.overlayMenu.cancelButton.hide();
+                AbstractDungeon.previousScreen = AbstractDungeon.screen;
+            }
+
+            AbstractDungeon.getCurrRoom().phase = AbstractRoom.RoomPhase.INCOMPLETE;
+            AbstractDungeon.gridSelectScreen.open(nonExertCards, 1, DESCRIPTIONS[1] + this.name + LocalizedStrings.PERIOD, false, false, false, false);
+        }
+    }
+    @Override
+    public void onUnequip() {
+        if (this.card != null) {
+            AbstractCard cardInDeck = AbstractDungeon.player.masterDeck.getSpecificCard(this.card);
+            if (cardInDeck != null) {
+                inBottledBurden.set(card, false);
+            }
+        }
+    }
+
+    @Override
+    public void update() {
+        super.update();
+        if (!this.cardSelected && !AbstractDungeon.gridSelectScreen.selectedCards.isEmpty()) {
+            this.cardSelected = true;
+            this.card = AbstractDungeon.gridSelectScreen.selectedCards.get(0);
+            inBottledBurden.set(card, true);
+            AbstractDungeon.getCurrRoom().phase = AbstractRoom.RoomPhase.COMPLETE;
+            AbstractDungeon.gridSelectScreen.selectedCards.clear();
+            this.description = String.format(DESCRIPTIONS[2], EXHUME_TURN, card.name);
+            this.tips.clear();
+            this.tips.add(new PowerTip(this.name, this.description));
+            this.initializeTips();
```

Comment:
This appears to be exactly `setDescriptionAfterLoading()`. Can we reuse that?

Reply:
Nice find. Thank you.